INSTALL.rst

```diff
@@ -15,16 +15,6 @@ this:
 
    $ pip install invenio-rest[cors]
 
-Configuration
--------------
-
-===================== =================================================
-`REST_ENABLE_CORS`    Set to ``True`` to enable Cross-Origin Resource
-                      Sharing. Defaults to ``False``.
-`REST_CORS_RESOURCES` Dictionary for configuring CORS for endpoints.
-                      See Flask-CORS for further details.
```

Comment:
You're right, actually ``REST_CORS_RESOURCES`` is wrong. It should be ``CORS_RESOURCES`` see https://flask-cors.readthedocs.io/en/latest/api.html#flask_cors.CORS and https://github.com/corydolphin/flask-cors/blob/master/flask_cors/core.py#L33